packages/react-button/src/components/CompoundButton/CompoundButton.stories.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+import { CompoundButton } from './CompoundButton';
+import { CompoundButtonProps } from './CompoundButton.types';
+import * as classes from '../Button/Button.stories.scss';
+
+/**
+ * Temporary Stack until there's one in its own package.
+ */
+const Stack = (props: React.PropsWithChildren<{ horizontal?: boolean }>) => {
+  const { horizontal, ...rest } = props;
+
+  return <div {...rest} className={horizontal ? classes.hStack : classes.vStack} />;
+};
+
+/**
+ * Temporary Text until there's one in its own package.
+ */
+// eslint-disable-next-line jsx-a11y/heading-has-content -- content passed via children
+const Text = (props: React.PropsWithChildren<{}>) => <h2 {...props} className={classes.text} />;
+
+const CompoundButtonVariants = (props: CompoundButtonProps) => (
+  <Stack horizontal>
+    <CompoundButton {...props} icon="O" secondaryText="This is some secondary text">
+      Hello, world
+    </CompoundButton>
+    <CompoundButton {...props} primary icon="X" secondaryText="This is some secondary text">
```

Comment:
if "content" is the slot for primary content, shouldn't this be `secondaryContent`?

Reply:
`children` is the slot for primary content right now but you're right, I'll make the change to `secondaryContent` since it better fits our slot approach.